public/app/percona/shared/components/PerconaBootstrapper/PerconaBootstrapper.tsx

```diff
@@ -81,8 +85,18 @@ export const PerconaBootstrapper = () => {
     }
   }, [dispatch, isLoggedIn, setSteps]);
 
+  const { result } = useSelector(getPerconaSettings);
+  const isTelemetryEnabled = !!result?.telemetryEnabled;
+  let telemetry = null;
```

Comment:
Not needed, we can go with conditional rendering, as below.
If we really want to return null, it should be `Telemetry`'s job to check the settings and render null, not the parent